packages/utils/src/lib/report-to-md.ts

```diff
@@ -248,16 +252,22 @@ function reportToAuditsSection(report: ScoredReport): string {
   return h2('🛡️ Audits') + NEW_LINE + NEW_LINE + auditsData;
 }
 
-function reportToAboutSection(report: ScoredReport): string {
+function reportToAboutSection(
+  report: ScoredReport,
+  commitData: CommitData | null,
+): string {
   const date = new Date().toString();
   const { duration, version, plugins, categories } = report;
-  // TODO: replace mock commitData with real data, ticket #192
-  const commitData =
-    '_Implement todos list_ ([3ac01d1](https://github.com/flowup/todos-app/commit/3ac01d192698e0a923bd410f79594371480a6e4c))';
+  const commitInfo = commitData
+    ? `${commitData.message} ([${commitData.hash.slice(
+        0,
+        7,
+      )}](${GITHUB_CLI_REPO_LINK}/commit/${commitData.hash}))`
```

Comment:
Without the link, please. We definitely don't want to hardcode our own repo, the commit is in the user's repo. Detecting the remote repo is part of #149, though.

```suggestion
    ? `${commitData.message} (${commitData.hash.slice(0, 7)})`
```